packages/desktop/electron/lib/aboutPreload.js

```diff
@@ -1,4 +1,16 @@
-const { contextBridge } = require('electron')
+const { contextBridge, ipcRenderer } = require('electron')
 const { version, build: { productName } } = require('../../package.json')
 
-contextBridge.exposeInMainWorld('app', { appName: productName, version, iconPath: './assets/logos/firefly_logo.svg' })
\ No newline at end of file
+contextBridge.exposeInMainWorld('about', {
+    getData: (callback) => {
+        ipcRenderer.invoke('menu-data').then(data => {
+            const aboutData = {
+                appName: productName,
+                version: data.strings.version.replace('{version}', version),
+                iconPath: './assets/logos/firefly_logo.svg'
+            }
+            
+            callback(aboutData)
+        })
+    }
+})
```

Comment:
I think we can do this without the callback:

```suggestion
    getData: () => ipcRenderer.invoke('menu-data').then((data) => {
            const aboutData = {
                appName: productName,
                version: data.strings.version.replace('{version}', version),
                iconPath: './assets/logos/firefly_logo.svg'
            }
            
            return aboutData
        })
    }
)
```

Reply:
Agreed, this was a bit legacy, from trying to get things to update!